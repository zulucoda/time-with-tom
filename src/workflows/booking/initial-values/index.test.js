import initialValues from '.';

describe('Workflows - Booking - Initial Values - Unit Test', () => {
  test('should return initial values', () => {
    const act = initialValues;
    expect(act).toMatchSnapshot();
  });
});
