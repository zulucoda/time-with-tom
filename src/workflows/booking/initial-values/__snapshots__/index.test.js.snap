// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Workflows - Booking - Initial Values - Unit Test should return initial values 1`] = `
Object {
  "date": 2020-04-21T00:00:00.000Z,
  "endTime": 2020-04-21T00:00:00.000Z,
  "startTime": 2020-04-21T00:00:00.000Z,
}
`;
